woocommerce/wc-admin
# Hook up import/totals endpoint to Historical Data Import screen

Part of #1851.

Hooks up the `/wc/v4/reports/import/totals` endpoint with the Historical Data Import page.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/57534196-cb276000-733f-11e9-875f-32edd6c15fd4.png)

### Detailed test instructions:
- ~First you will need to merge the branch from #2189 (git merge add/1850-import-status).~ (no longer necessary)
- In `client/analytics/settings/historical-data/layout.js` line 70 set: `const showImportStatus = true;`, so progress bars are displayed.
- Go to _Settings_ and scroll down to _Import Historical Data_.
- Verify there are numbers next to the progress bars.
- Change the number of days to import and toggle the 'skip imported' checkbox and verify the numbers update accordingly.
